src/dali/abstract_pair_converter_plugin.py

```diff
@@ -87,3 +117,109 @@ def get_conversion_rate(self, timestamp: datetime, from_asset: str, to_asset: st
             )
 
         return result
+
+    def _build_fiat_list(self) -> None:
+        try:
+            response: Response = self.__session.get(_EXCHANGE_SYMBOLS, timeout=self.__TIMEOUT)
+            # {
+            #     'motd':
+            #         {
+            #             'msg': 'If you or your company ...',
+            #             'url': 'https://exchangerate.host/#/donate'
+            #         },
+            #     'success': True,
+            #     'symbols':
+            #         {
+            #             'AED':
+            #                 {
+            #                     'description': 'United Arab Emirates Dirham',
+            #                     'code': 'AED'
+            #                 },
+            #             ...
+            #         }
+            # }
+            data: Any = response.json()
+            if data[_SUCCESS]:
+                self.__fiat_list = [fiat_iso for fiat_iso in data[_SYMBOLS] if fiat_iso != "BTC"]
+            else:
+                if "message" in data:
+                    LOGGER.error("Error %d: %s: %s", response.status_code, _EXCHANGE_SYMBOLS, data["message"])
+                response.raise_for_status()
+
+        except JSONDecodeError as exc:
+            LOGGER.debug("Fetching of fiat symbols failed. The server might be down. Please try again later.")
+            raise Exception("JSON decode error") from exc
+
+    def _add_fiat_graph_to(self, graph: Dict[str, List[str]], markets: Dict[str, List[str]]) -> None:
```

Comment:
The name of this function is a bit unclear: did you mean `_add_to_fiat_graph`? The trailing `to` is a little confusing.

Reply:
Well, this function adds the fiat graph to the first argument of the function (the graph of crypto assets). The function adds edges one by one to the crypto asset graph; the fiat graph is never really made actually, so it should probably be more like `_add_fiat_edges_to`? What do you think is more readable? or is there some kind of naming convention I'm abusing?

Reply:
Thanks for the clarification. The idea behind naming functions in RP2 and Dali is to spell out their action as clearly as possible in English (inspired by the [GNU Coding Standard](https://www.gnu.org/prep/standards/html_node/Names.html#Names)). In this case I would suggest something like `_add_fiat_edges_to_graph`.